src/system/unwind.c

```diff
@@ -114,6 +114,7 @@ _Unwind_Reason_Code trace_fn(_Unwind_Context* unwind_ctx, void* d) {
 /******************************************************************************/
 // recover registers from a data abort. Specific knowledge about callee stack
 // used from FreeRTOS_DataAbortHandler and it calling DataAbortInterrupt
+__attribute__((target("arm")))
```

Comment:
Probably best to document why this attribute is necessary

Reply:
Done in https://github.com/purduesigbots/pros/pull/708/commits/e2fcbd135d52d0829681ea946de8dd66e3739697.